src/frontend/apps/lti_site/i18n/fr_FR.json

```diff
@@ -23,26 +23,54 @@
     "description": "Title for picture in picture switch button when player is the main element",
     "message": "Afficher le document"
   },
-  "apps.bbb.SelectContent.addMeeting": {
-    "description": "Text displayed on a button to add a new meeting.",
+  "apps.bbb.SelectContent.addClassroom": {
+    "description": "Text displayed on a button to add a new classroom.",
     "message": "Ajouter une classe virtuelle"
   },
-  "apps.bbb.SelectContent.loadingMeetings": {
-    "description": "Accessible message for the spinner while loading the meetings in lti select view.",
+  "apps.bbb.SelectContent.loadingClassrooms": {
+    "description": "Accessible message for the spinner while loading the classrooms in lti select view.",
     "message": "Chargement des classes virtuelles..."
   },
   "apps.bbb.SelectContent.notStarted": {
-    "description": "Text helper displayed if a meeting is not started.",
+    "description": "Text helper displayed if a classroom is not started.",
     "message": "Non démarrée"
   },
   "apps.bbb.SelectContent.select": {
-    "description": "Accessible message for selecting a meeting.",
+    "description": "Accessible message for selecting a classroom.",
     "message": "Sélectionner {title}"
   },
   "apps.bbb.SelectContent.started": {
-    "description": "Text helper displayed if a meeting is started.",
+    "description": "Text helper displayed if a classroom is started.",
     "message": "Démarrée"
   },
+  "apps.deposit.SelectContent.addFileDepository": {
+    "description": "Text displayed on a button to add a new fileDepository.",
+    "message": "Ajouter un dépôt de fichiers"
+  },
+  "apps.deposit.SelectContent.loadingFileDepositories": {
+    "description": "Accessible message for the spinner while loading the fileDepositories in lti select view.",
```

Comment:
file depositories?
and ditto below occurences

Reply:
Fixed